docs/openapi/components/schemas/common/FSMAGrowingCTE.yml

```diff
@@ -0,0 +1,84 @@
+$linkedData:
+  term: FSMAGrowingCTE
+  '@id': https://w3id.org/traceability#FSMAGrowingCTE
+title: FSMA Growing CTE
+description: >-
+  "Information regarding an FSMA Growing CTE (Critical Tracking Event), designed to meet reporting
+  requirements for the FDA Traceability Rule as specified here:
+  https://www.fda.gov/food/food-safety-modernization-act-fsma/fsma-proposed-rule-food-traceability"
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - FSMAGrowingCTE
+      - type: string
+        const: FSMAGrowingCTE
+  traceabilityLot:
+    title: Traceability Lot
+    description: The lot of food being tracked.
+    $ref: ./FSMATraceabilityLot.yml
+    $linkedData:
+      term: traceabilityLot
+      '@id': https://w3id.org/traceability#FSMATraceabilityLot
+  growingAreaCoordinates:
+    title: Growing Area Coordinates [TODO this could be attached to lot directly removing need for this schema - but should it? Won't be used by other schemas that ref trace lot... I think?]
+    description: Coordinates of the area the product was grown.
+    $ref: ./GeoCoordinates.yml
+    $linkedData:
+      term: growingAreaCoordinates
+      '@id': https://schema.org/GeoCoordinates
+  additionalData:
+    title: Additional Data
+    description: Additional Key Data Elements (KDEs).
+    type: array
+    items:
+      $ref: ./FSMAAbstractKDE.yml
+    $linkedData:
+      term: additionalData
+      '@id': https://w3id.org/traceability#FSMAAbstractKDE
+additionalProperties: false
+example: |-
+  {
+    "type": "FSMAGrowingCTE",
+    "traceabilityLot": {
+      "type": "FSMATraceabilityLot",
+      "lotCode": "CHE-MIG-TTF1-061321-H37J",
+      "lotCodeAssignmentMethod": "Lot codes are assigned based on the commodity, variety, location, harvest date and a random 4 digit string.",
+      "lotCodeGeneratorLocation": {
+        "type": [
+          "Place"
+        ],
+        "address": {
+          "type": [
+            "PostalAddress"
+          ],
+          "organizationName": "John's Produce",
+          "streetAddress": "4335 Walsh Underpass",
+          "addressLocality": "Port Mark",
+          "addressRegion": "LA",
+          "postalCode": "67391",
+          "addressCountry": "United States"
+        },
+        "locationName": "John's Tomato Farm #1"
+      },
+      "lotCodeGeneratorPOC": {
+        "type": "Entity",
+        "entityType" : "Person",
```

Comment:
You are gonna end up with a bunch of undefined terms... thats going to be data that is not processable or mergeable...

Reply:
@OR13 This example text represents a reference to our current Entity schema. I'm not sure I'm following, what about that schema will result in undefined terms?